src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php

```diff
@@ -63,7 +63,7 @@ public function exists($name)
      *
      * @throws \Twig_Error_Loader if the template could not be found
      */
-    protected function findTemplate($template)
+    protected function findTemplate($template, $throw = true)
     {
         $logicalName = (string) $template;
```

Comment:
Not really. If I do that, it breaks elsewhere. The problem is that Symfony uses objects and Twig do not support them anymore in 2.0.